### ISSUE TITLE
Accessibility: TAB into keyboard help screen should focus on close button

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4476]
### Is it an issue related to Adaptive Cards?

- [ ] Yes, this is an Adaptive Card issue but it is specific to Web Chat.

### Do you have any screenshots?

![image](https://user-images.githubusercontent.com/1622400/198818026-9ecdbbbd-da59-4208-8408-ac2231a45d1b.png)

### What version of Web Chat are you using?

Latest production

### Which distribution are you using Web Chat from?

Unrelated

### Which hosting environment does this issue primarily affect?

Web apps

### Which browsers and platforms do the issue happened?

Browser: Edge (latest), Browser: Chrome (latest), Browser: Firefox (latest), Browser: Safari (latest), Browser: IE Mode (latest), Platform: Windows, Platform: macOS

### Which area does this issue affect?

Others/unrelated

### Is this an accessibility issue?

- [X] Yes, this is an accessibility issue.

### Please describe the bug

When press <key>TAB</key> to focus on Web Chat, a keyboard help screen will be shown.

Although it looks like the focus is on the close button, the actual focus is on the dialog itself.

It should focus on the close button. However, we might have some reasons on why we are doing this.

### How to reproduce the issue?

1. Open Web Chat Loader
2. Press <key>TAB</key>

### What is the expected and actual behavior?

Expected:

The focus (`document.activeElement`) should be on the close button, which has the following DOM:

```html
<button aria-label="Close" class="webchat__keyboard-help__close-button" tabindex="-1" type="button">
```

Actual:

The focus (`document.activeElement`) is on the dialog DOM element:

```html
<div aria-hidden="false" class="webchat__keyboard-help webchat--css-xpubm-1kp5mon webchat__keyboard-help--shown" role="dialog" tabindex="0">
```

### Additional context

@beyackle2 this is the bug you mentioned this week
